docs/editors/sublime.md

```diff
@@ -86,6 +86,17 @@ This paragraph contains a few tips & trick that can improve your daily productiv
 
 ### Optional LSP client tweaks
 
+If you want Metals to automatically organize the imports on file save set the following setting in the "Preferences > Preferences: LSP Settings":
+
+```json
+{
+  // ...
+  "lsp_code_actions_on_save": {
+    "source.organizeImports": true
```

Comment:
unfortunately this will not work as expected.

Metals currently needs the file to have been compiled before being able to run Organize Imports, so this will go as follow:

- save
- get a warning about the file not being saved (confusing, I know, see https://github.com/scalameta/metals/issues/2100)
- wait for compilation to end
- save again
- now it works

See the linked issue for a more in-depth discussion

Reply:
Hmm I see. So to get the feature working with the current Metals limitation the code action needs to be somehow exposed to the client like in Vim?

Reply:
`Currently it seems that LSP supports source.organizeImports only as code action on save.`
LSP in sublime? LSP specification defines it as a code action available anywhere.
You can simply bind it to shortcut and execute anytime you want at least in emacs/VSCode.

Reply:
We might need to fix #2100 first I think before this will work sublime then.

Reply:
OK I will add specific command to invoke `source.organizeImports` manually. 
But why isn't this code action available when invoking the general code action next to imports ?

Reply:
we have to start with something, we can definitely improve it